src/main/java/org/spongepowered/api/Game.java

```diff
@@ -22,7 +22,6 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-
```

Comment:
The Google Java style requires an empty line between the license header and the package statement, but I don't think it's that important: https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s3-source-file-structure